.github/workflows/build.yml

```diff
@@ -344,6 +344,8 @@ jobs:
 
       - name: Unit tests
         shell: bash
+        env:
+          NODE_OPTIONS: --experimental-vm-modules
```

Comment:
why do we need this?

Reply:
Presumably fixing https://github.com/renovatebot/renovate/pull/23627#issuecomment-1658328852

Reply:
ah, right. prettier is esm and has a simple cus wrapper with dynamic import.